src/loader.js

```diff
@@ -88,22 +88,26 @@
         return entry.module;
       });
 
+    // @ifdef STRICT
```

Comment:
Yeah I started out with uglifyjs' preprocessor removing out the strict comparison stuff. But then I realized that since we don't run uglify on the `.src.js` file that uglify wouldn't be a good option for [`es6-module-loader.src.js`](https://github.com/ModuleLoader/es6-module-loader/blob/master/dist/es6-module-loader.src.js), but only for `es6-module-loader.js`.

I'll look into maybe some other options that will preprocess out the strict code that becomes dead when the global variable is set.